### PR TITLE
Pricing Page

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**

- Page available at `/pricing`
- Added 3 new types of `Blockquotes` that includes the image of the speaker.

This PR fixes #

**Notes for Reviewers**
- Couldn't fix the responsiveness of the Quotes section yet. Working on it.
- The content in cards, table, FAQs will be revised soon.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6f76f01a446f74f5c929f9b33e03f1b7e58e2c87 at: https://62a65e42863c2a2b6670fea9--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1f35f041c9c92b6f50b7fa837715584dc696b18b at: https://62a66509c9842332d2fa3fab--layer5.netlify.app

**Comment by leecalcote** [comment]
Feedback:

1. Add Monthly | Yearly price switcher (in another PR)
2. Make quotes responsive (screenshot below)
3. Make feature checkmarks expandable to show a couple of lines of more details. Screen Shot 2022-06-12 at 8 24 42 PM

**Comment by l5io** [comment]
🚀 Preview for commit da2bc85b4be06ee30741b5622a4a2c05360726a5 at: https://62a6ad4ca32b4d61aab0e07e--layer5.netlify.app

File: src/sections/Pricing/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { FiCheck } from "@react-icons/all-files/fi/FiCheck";
+
+
+const Pricing = () => {
+
+  return (
+    <PricingWrapper>
+
+      <div className="headers">
+        <h1 className="header-heading">Plans or Every Team Size</h1>
```
Comment (leecalcote) [inline src/sections/Pricing/index.js:18]:
```suggestion
        <h1 className="header-heading">Plans For Every Team Size</h1>
```

**Comment by l5io** [comment]
🚀 Preview for commit 9e7af4231fc77ddc3b0ba412792febad5618dc76 at: https://62a724aa2040123256f5c72e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 95f00d5a8ea22fa8c46eed8baeee8e75ddbf8f2a at: https://62a904781c9a4c255c2b4b77--layer5.netlify.app

**Comment by warunicorn19** [comment]
Coming soon button should be disabled right?

**Comment by l5io** [comment]
🚀 Preview for commit 067486950ae97338d8a8ff764bf3a1f8cfc10fbc at: https://62a9367ee3e65a1c562576f0--layer5.netlify.app

**Comment by leecalcote** [comment]
> Coming soon button should be disabled right?

Yes, please.

**Comment by l5io** [comment]
🚀 Preview for commit 62228471b5a68471ae9345c490b783cb1ad7b404 at: https://62aadf5a35ec4d4dcd1f5172--layer5.netlify.app

**Comment by debo19** [comment]
Switcher is done, just need to change the review cards styling to gel with the page style and change logos to company logos.

**Comment by leecalcote** [comment]
Coming Soon button should be disabled. 

Subscription duration buttons has an active and / or down state the confuses slightly. 


https://user-images.githubusercontent.com/7570704/174053532-15871b7e-bf7f-43db-bd77-73c5fdf72b80.MOV

**Comment by l5io** [comment]
🚀 Preview for commit 3769d431c7e8a6b6508761f595791301ebc1367c at: https://62abac95297a5f4272c0eac1--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0983118289e861814a5a7627b453e33c3b16030c at: https://62acacd219bcdd0cb00a6abc--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2128437d46650885a7d4672c3525d94e494845e3 at: https://62acb2abe74c6311a9188cbb--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 076d28ee6164363d1b98e67d8b68bea83c30824e at: https://62ace808d87dbc26a57c63a2--layer5.netlify.app

**Comment by leecalcote** [comment]
@debo19 are the two profile photos to be used elsewhere in the site? If not, please move them into the pricing page and not kept separately. Separately kept files are likely to be orphaned. William's last name is Morgan.

**Comment by l5io** [comment]
🚀 Preview for commit e1185493714db3f386cf54762f76362f99ec8854 at: https://62afa524e39add1e1f9830ad--layer5.netlify.app

**Comment by leecalcote** [comment]
Great job on the categories and their icons, too. A bit more data needed (from me), but this is mergable as-is.

**Pull request merged**